roles/bootstrap-os/tasks/main.yml

```diff
@@ -16,4 +16,10 @@
   register: ostree
 
 - set_fact:
-    is_atomic: "{{ ostree.stat.exists }}"
\ No newline at end of file
+    is_atomic: "{{ ostree.stat.exists }}"
+
+- name: Assign inventory name to unconfigured hostnames
+  hostname:
+    name: "{{inventory_hostname}}"
+  when: ansible_hostname == 'localhost'
```

Comment:
Fixed, gather_subset can use only few categories, !all collect minimum amount of data